google/cloud/bigtable/examples/table_admin_snippets.cc

```diff
@@ -694,7 +696,11 @@ void WaitForConsistencyCheck(
     fut.get();  // simplify example by blocking until operation is done.
   }
   //! [wait for consistency check]
-  (std::move(old_admin), argv.at(2));
+  (old_admin, argv.at(2));
```

Comment:
Consider creating a new function that receives `old_admin` as a parameter, but extends its lifetime until after the futures are destroyed.

Something like:

```cc
void WaitForConsistencyCheckImpl(
    google::cloud::bigtable::TableAdmin const& admin,
   std::vector<std::string> const& argv) { /* current code here */ }

void WaitForConsistencyCheck(
    google::cloud::bigtable_admin::BigtableTableAdminClient const&,
    std::vector<std::string> const& argv) {
  auto old_admin = google::cloud::bigtable::TableAdmin(
      google::cloud::bigtable::MakeAdminClient(argv.at(0)), argv.at(1));
  WaitForConsistencyCheckImpl(old_admin, argv);
}
```

Reply:
This does not fix the problem. It appears that `old_admin` needs to exist for a certain amount of time. Not just exist for the scope of the method.